<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoSubmit</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    font-size: 16;
    font-family: 'Cascadia Code PL', consolas, '微软雅黑';
    padding: 5px;
}
</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="main">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0,2">
    <item>
     <layout class="QHBoxLayout" name="control">
      <item>
       <widget class="QDateTimeEdit" name="date">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2020</year>
          <month>11</month>
          <day>23</day>
         </datetime>
        </property>
        <property name="maximumDateTime">
         <datetime>
          <hour>23</hour>
          <minute>59</minute>
          <second>59</second>
          <year>2050</year>
          <month>12</month>
          <day>31</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2020</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="minimumDate">
         <date>
          <year>2020</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::DaySection</enum>
        </property>
        <property name="displayFormat">
         <string>yyyy/MM/dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="currentSectionIndex">
         <number>2</number>
        </property>
        <property name="timeSpec">
         <enum>Qt::LocalTime</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_select">
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addr">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>请选择一个有效的 Json 文件...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_start">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="log_local">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cascadia Code PL','consolas','微软雅黑'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;-- LOCAL LOGS --&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progress_bar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="log_network">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>240</height>
       </size>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cascadia Code PL','consolas','微软雅黑'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;-- NETWORK LOGS --&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="create_txt"/>
    <addaction name="convert"/>
    <addaction name="separator"/>
    <addaction name="quit"/>
   </widget>
   <widget class="QMenu" name="config">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="common"/>
    <addaction name="separator"/>
    <addaction name="name"/>
    <addaction name="rank"/>
    <addaction name="separator"/>
    <addaction name="url"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="about"/>
    <addaction name="about_qt"/>
   </widget>
   <addaction name="file"/>
   <addaction name="config"/>
   <addaction name="help"/>
  </widget>
  <action name="about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="about_qt">
   <property name="text">
    <string>关于 Qt</string>
   </property>
  </action>
  <action name="create_txt">
   <property name="text">
    <string>新建 txt</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="convert">
   <property name="text">
    <string>转换为 Json</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="name">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>校正名称</string>
   </property>
  </action>
  <action name="rank">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>标注评级</string>
   </property>
  </action>
  <action name="url">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>编辑 URL 规则</string>
   </property>
  </action>
  <action name="common">
   <property name="text">
    <string>常规设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>date</tabstop>
  <tabstop>btn_select</tabstop>
  <tabstop>addr</tabstop>
  <tabstop>btn_start</tabstop>
  <tabstop>log_local</tabstop>
  <tabstop>log_network</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
